tox.ini

```
@@ -1,10 +1,14 @@
 [tox]
 envlist =
-    py37-django{111,22}
+    py{38,39}-django{30,31,32,40,41,42}
 
 [testenv]
 commands = python manage.py test app
 deps =
-    django111: Django>=1.11, <1.12
-    django22: Django>=2.2, <2.3
+    django30: Django>=3.0, <3.1
+    django31: Django>=3.1, <3.2
+    django32: Django>=3.2, <3.3
+    django40: Django>=4.0, <4.1
+    django41: Django>=4.1, <4.2
+    django42: Django>=4.2, <4.3
```

Does adding these break the test suite?
```suggestion
    django42: Django>=4.2, <4.3
    django50: Django>=5.0, <5.1
    django51: Django>=5.1, <5.2
```

---

I'd prefer not to update this project for Django 5.2 until we start upgrading the main application to Django 5.2.